docusaurus/docs/iOS/basics/integration.md

```diff
@@ -0,0 +1,184 @@
+![Screenshot shows an Xcode screen selecting a dependency version and an Xcode screen downloading that dependency](../assets/spm-01.png)
+
+Based on the repository you select you can find 3 different targets: StreamChat, StreamChatUI and StreamChatSwiftUI.
+
+- If you want to use the UIKit components, select **StreamChatUI**.
+- If you want to use the SwiftUI components, select **StreamChatSwiftUI**.
+- If you don't need any UI components, select **StreamChat**.
+
+![Screenshot shows an Xcode screen with dependency targets to be selected](../assets/spm-02.png)
+
+After you press finish, it's done!
+
+:::caution
+Because StreamChat SDKs have to be distributed with its resources, the minimal Swift version requirement for this installation method is 5.3. If you need to support older Swift version, please install it using CocoaPods.
+:::
+
+_More information about Swift Package Manager [can be found here](https://www.swift.org/package-manager/)_
+
+### CocoaPods
+
+In your project's `Podfile`, add one of these options
+
+- For the LLC (**StreamChat**) use:
+  - `pod 'StreamChat', '~> 4.0.0'`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `pod 'StreamChatUI', '~> 4.0.0'`
+- For the SwiftUI components (**StreamChatSwiftUI**, which depends on **StreamChat**) use:
+  - `pod 'StreamChatSwiftUI', '~> 4.0.0'`
+
+<details><summary>➤ Do you want to use <b>XCFrameworks</b>? (Click to read more)</summary>
+<p>
+
+:::caution
+Our XCFrameworks are built with **Swift 5.5**. In order to use them you need **Xcode 13** or above
+:::
+
+You can learn more about [our Module Stable XCFrameworks here](#xcframeworks)
+
+- For the LLC (**StreamChat**) use:
+  - `pod 'StreamChat-XCFramework', '~> 4.0.0'`
+- For the UIKit components (**StreamChatUI**, which depends on **StreamChat**) use:
+  - `pod 'StreamChatUI-XCFramework', '~> 4.0.0'`
+
+</p>
+</details>
+
+Example: If you want to use the UIKit components, it should look similar the snippet below.
+
+```ruby
+target 'MyProject' do
+  use_frameworks!
+
+  # Pods for MyProject
+  pod 'StreamChatUI', '~> 4.0.0'
+end
+```
+
+StreamChatUI and StreamChatSwiftUI pod will automatically include the StreamChat dependency. If you want just the StreamChat dependency, without the UI components, use StreamChat as shown above.
+
+Now that we’ve modified our Podfile, let’s go ahead and install the project dependencies via the terminal with one simple command:
+
+```bash
+pod install --repo-update
+```
+
+The above command will generate the **MyProject.xcworkspace** file automatically.
+
+With our workspace now containing our Pods project with dependencies, as well as our original project, let’s go ahead and move over to Xcode to complete the process.
+
+_More information about CocoaPods [can be found here](https://cocoapods.org/)_
+
+### Carthage
+
+In your project's `Cartfile`, add one of these options
+
+- For the LLC (**StreamChat**) use:
+  - `github "getstream/stream-chat-swift" ~> 4.0.0`
```

Comment:
@polqf It would be safer to call out v4.6.0 as the first fully supported version here.

```suggestion
  - `github "getstream/stream-chat-swift" ~> 4.6.0`
```